### PR TITLE
KOGITO-8802: SWF Diagram Editor - depiction of actionMode parameter

### PR DESCRIPTION
**Pull request by hasys**
Hello @romartin, @LuboTerifaj,

there is a PR with actionMode depiction.

I reused depiction from BPMN editor:
image

Thank you!

### CODE REVIEW

**Review by LuboTerifaj — APPROVED** [review]
It looks good to me. Thanks!

File: packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-client/src/main/java/org/kie/workbench/common/stunner/sw/client/shapes/icons/BottomDepiction.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        setListening(true);
+
+        MultiPath multiPath = new MultiPath(icon)
+                .setStrokeWidth(1)
+                .setStrokeColor("#4F5255")
+                .setFillColor("#4F5255")
+                .setListening(false);
+        add(multiPath);
+
+        mouseClickHandler = border.addNodeMouseClickHandler(
```
Comment (romartin) [inline packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-client/src/main/java/org/kie/workbench/common/stunner/sw/client/shapes/icons/BottomDepiction.java:53]: is this handler regsitration being destroyed at the end?

Comment (hasys) [reply packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-client/src/main/java/org/kie/workbench/common/stunner/sw/client/shapes/icons/BottomDepiction.java:53]: Hey @romartin, sure. It is removed in ´destroý method.

**Comment by hasys** [comment]
Hi @romartin, rebased and ready for re-review. Thank you!

File: packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-client/src/main/java/org/kie/workbench/common/stunner/sw/client/shapes/icons/BottomDepiction.java (hunk 1 above)
Comment (romartin) [reply packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-client/src/main/java/org/kie/workbench/common/stunner/sw/client/shapes/icons/BottomDepiction.java:53]: sorry, thanks!

**Review by romartin — APPROVED** [review]

**Pull request merged**